src/sentry/static/sentry/app/components/group/externalIssueForm.jsx

```diff
@@ -201,6 +201,28 @@ export class SentryAppExternalIssueForm extends React.Component {
     this.props.onSubmitSuccess(issue);
   };
 
+  onSubmitError = () => {
+    const appName = this.props.sentryAppInstallation.sentryApp.name;
+    addErrorMessage(t(`Unable to ${this.props.action} ${appName} issue.`));
+  };
+
+  getFieldDefault(field) {
+    const {group, sentryAppInstallation} = this.props;
+    if (field.type == 'textarea') {
+      field.maxRows = 10;
+      field.autosize = true;
+    }
+    switch (field.default) {
+      case 'issue.title':
+        return group.title;
+      case 'issue.description':
+        return `Sentry Issue: [${group.shortId}](${group.permalink}?referrer=${sentryAppInstallation
```

Comment:
Will `group.permalink` ever have query params? If so, the additional `?` will break things. Should we use something like `qs` here?